src/Tasks/Microsoft.NET.Build.Tasks/DependencyContextBuilder.cs

```diff
@@ -32,7 +32,7 @@ public DependencyContextBuilder(SingleProjectInfo mainProjectInfo, ProjectContex
             _projectContext = projectContext;
 
             // This resolver is only used for building file names, so that base path is not required.
-            _versionFolderPathResolver = new VersionFolderPathResolver(path: null);
+            _versionFolderPathResolver = new VersionFolderPathResolver(null);
```

Comment:
Actually no it's fixed in master but by using the new named parameter rootPath. Can you switch to that so that it's more self documenting?